src/clientversion.h

```diff
@@ -38,7 +38,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The " PACKAGE_NAME " Developers"
```

Comment:
Concept ACK, haven't had a chance to test.

On Tuesday, 22 December 2015, Mark Friedenbach <notifications@github.com>
wrote:

> In src/clientversion.h
> <https://github.com/bitcoin/bitcoin/pull/7192#discussion_r48238911>:
>
> > @@ -38,7 +38,7 @@
> >  #define DO_STRINGIZE(X) #X
> >
> >  //! Copyright string used in Windows .rc files
> > -#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
> > +#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The " PACKAGE_NAME " Developers"
>
> As a separate issue from Jonas, are we sure the copyright name is
> something we want to change? I've had to deal with this as maintainer of
> Freicoin .. changing the product name would result in "Copyright 2009-2015
> The Freicoin Core Developers" which would certainly not be okay (or legal?).
>
> It would be nice to detect if PACKAGE_NAME is changed from the default,
> and if so add a another copyright string (while maintaining the hard-coded
> "Bitcoin Core" text).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/7192/files#r48238911>.
>